evaluator/builtin_string_test.go

```diff
@@ -707,15 +707,18 @@ func (s *testEvaluatorSuite) TestRpad(c *C) {
 		{"hi", 5, "?", "hi???"},
 		{"hi", 1, "?", "h"},
 		{"hi", -1, "?", ""},
-		{"hi", 1, "", ""},
+		{"hi", 1, "", "h"},
+		{"hi", 5, "", ""},
+		{"hi", 5, "ab", "hiaba"},
+		{"hi", 6, "ab", "hiabab"},
```

Comment:
The test can't distinguish null and empty string.
`rpad("hi", 0, "?")` is empty string, not NULL.